test/fork/plugin/onStreamCanceled.t.sol

```diff
@@ -9,7 +9,7 @@ import { Fork_Test } from "../Fork.t.sol";
 
 import { Permit2Params } from "src/types/Permit2.sol";
 
-/// @dev Runs against multiple assets.
+/// @dev Runs against the asset passed to its constructor.
```

Comment:
Yes, and there are multiple instances of this contract. I'll rewrite the comment to say this:

> Runs against multiple fork assets